controllers/fenceagentsremediation_controller.go

```diff
@@ -74,39 +75,70 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 	emptyResult := ctrl.Result{}
 
 	// Fetch the FenceAgentsRemediation instance
-	far := &v1alpha1.FenceAgentsRemediation{}
+	far := &farV1alpha1.FenceAgentsRemediation{}
 	if err := r.Get(ctx, req.NamespacedName, far); err != nil {
 		if apiErrors.IsNotFound(err) {
-			// FAR is deleted, stop reconciling
-			r.Log.Info("FAR CR is deleted - nothing to do", "CR Name", req.Name, "CR Namespace", req.Namespace)
+			// Request object of FAR CR was not found, could have been deleted after reconcile request.
+			// Return and don't requeue
+			r.Log.Info("FenceAgentsRemediation CR was not found", "CR Name", req.Name, "CR Namespace", req.Namespace)
 			return emptyResult, nil
 		}
-		r.Log.Error(err, "failed to get FAR CR")
+		r.Log.Info("Failed to get FenceAgentsRemediation CR, requeuing.", "CR Name", req.Name)
```

Comment:
I think an error log was appropriate here, why change to Info ?

Reply:
Because we return the error on the next line. Otherwise, the log will have two errors (one from log and from return) for one error..

Reply:
Sorry I don't quite follow.
- Why returning the error should effect whether it should be logged or not ?
- And even if it does, you log it anyway, so why log an error as an info ?

Reply:
I was thinking of avoiding logging of errors before reconcile returns an error in order to avoid noisy reconcile (duplicating error), but it is better to duplicate rather just depends on errors from reconcile.
Fixed in https://github.com/medik8s/fence-agents-remediation/pull/55/commits/f1cec90b2a1e1151e7e432259a52efd677fb2bed of #55